int32 detection_mode
---
bool response